archivers/bzip3/Portfile

```diff
@@ -18,8 +18,8 @@ checksums           rmd160  3940e049433a348ccc9fe4f0fcc3fe76cbae37ad \
                     sha256  bba90d867f53efa4291de1df9c22da1578e1a93ca819e0fc68881da6fcc4149d \
                     size    399621
 
-# https://trac.macports.org/ticket/66644
-compiler.blacklist-append \
```

Comment:
then you'll likely don't need the PortGroup anymore either - right?

Reply:
Indeed, thank you. Will fix that.

Reply:
@reneeotten Fixed.